main/src/cgeo/geocaching/network/HtmlImage.java

```diff
@@ -232,7 +232,9 @@ public void run() {
                         final ImmutablePair<BitmapDrawable, Boolean> loaded = loadFromDisk();
                         final BitmapDrawable bitmap = loaded.left;
                         if (loaded.right) {
-                            emitter.onNext(bitmap);
+                            if (bitmap != null) {
```

Comment:
while this looks like an easy fix, I couldn't find a call chain that legitimately would lead  to a null bitmap with a true flag in this context.
So it might be worthwhile to look a little deeper into the root cause.